conf/nginx.conf

```diff
@@ -1,5 +1,4 @@
-location __PATH__ 
-{
+location __PATH__/ {
```

Comment:
Eeeeh can we add a sub_path_only redirect such that accessing `domain.tld/wekan` will still work instead of just `domain.tld/wekan/` (with a `/`) ? :stuck_out_tongue_winking_eye:

Reply:
```suggestion
#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
location __PATH__/ {
```